acc_test/resource_kafka_test.go

```diff
@@ -49,10 +56,25 @@ func TestKafkaUserResource(t *testing.T) {
 	invalidKafkaUserCreateConfigDuplicate := fmt.Sprintf(string(configBytes5), username, apiKey, hostname, kafkaNodeSize, kafkaVersion, zookeeperNodeSize,
 		kafkaUsername1, newPassword,
 		kafkaUsername2, newPassword,
-                kafkaUsername1, oldPassword)
+		kafkaUsername1, oldPassword)
 	invalidKafkaUserCreateConfig := fmt.Sprintf(string(configBytes4), username, apiKey, hostname, kafkaNodeSize, kafkaVersion, zookeeperNodeSize,
 		kafkaUsername3, oldPassword)
 
+	// Kafka topic management tests
+	createKafkaTopicConfig := fmt.Sprintf(string(configBytes6), username, apiKey, hostname, kafkaNodeSize, kafkaVersion, zookeeperNodeSize,
```

Comment:
Awesome, thanks for doing this. One question though - do all of the configs now have duplicated resource definitions for the same cluster? (i.e. if we ever update it, we need to update it in five places). If so, is it possible to restructure the data files so that it is only specified once?

Reply:
Yeah, all the tf files share the same cluster configurations. But the most important configs (the configs that might be changed) are all parameterised. See below:
```
	kafkaUsername1 := "charlie1"
	kafkaUsername2 := "charlie2"
	kafkaUsername3 := "charlie3"
	oldPassword := "charlie123!"
	newPassword := "charlie123standard!"
	kafkaNodeSize := "KFK-DEV-t4g.medium-80"
	zookeeperNodeSize := "KDZ-DEV-t4g.small-30"
	kafkaVersion := "apache-kafka:2.7.1.ic1"
	topic1 := "test1"
	topic2 := "test2"
```